### PR TITLE
Moved and revised the char article

### PR DESCRIPTION
**Pull request by pkulikov**
Contributes to #10726
Supported by dotnet/samples#1785

@BillWagner could you also please take a look at https://github.com/dotnet/docs/issues/10726#issuecomment-550174448, which is about how to further address #10726

### CODE REVIEW

File: docs/csharp/language-reference/builtin-types/char.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: "char type- C# reference"
+ms.custom: seodec18
```
Comment (BillWagner) [inline docs/csharp/language-reference/builtin-types/char.md]: This should be removed. We were tracking the results of one targeted push for SEO. This new article shouldn't be tracked the same.

**Review by BillWagner — CHANGES_REQUESTED** [review]
This is great @pkulikov 

There's one small suggestion, and then it's ready to :shipit:

Thanks for all your work in this area.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @pkulikov 
I'll :shipit: when the build finishes.

File: docs/csharp/language-reference/builtin-types/char.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/builtin-types/char.md]: @BillWagner that's clear: I've removed the line.

**Pull request merged**